gateway/1.4/modules/ROOT/pages/_partials/task-config-liveness-probe.adoc

```diff
@@ -0,0 +1,99 @@
+//tag::intro1[]
+
+Flex Gateway offers the following CLI command to test the health of a Flex Gateway:
+
+[source,ssh]
+----
+flexctl probe --check=liveness
+----
+
+This command returns either a status code of `0` for a healthy Flex Gateway or `1` for an unhealthy Flex Gateway.
+
+//end::intro1[]
+
+//tag::intro2[]
+By using the liveness check command, you can either configure a Kubernetes liveness probe or a Docker health check command to restart a failed Flex Gateway.
```

Comment:
A couple of things to here:
- From 1.4.4 we will configure a default liveness probe in kubernetes that can be customized by the users
- In docker, the health check only marks the container as healthy or unhealthy, but it doesn't restart it

```suggestion
By using the liveness check command, you can either customize the Kubernetes liveness probe provided by default to restart failed Flex Gateway pods or configure a Docker health check command.
```